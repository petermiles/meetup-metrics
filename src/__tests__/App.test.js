// External Dependencies
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from "enzyme";

// Internal Dependencies
import App from '../App';

// Local Variables
const app = shallow(<App />);

// Begin Tests
describe('App', () => {
  it('renders correctly', () => {
    expect(app).toMatchSnapshot();
  });
});
