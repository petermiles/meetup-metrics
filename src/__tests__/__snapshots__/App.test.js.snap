// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="app"
>
        <withStyles(TopNav) />
        <MembershipTrend />
        <withStyles(Attendance) />
        <iframe
                frameBorder="0"
                height="485"
                src="http://meetu.ps/3fFbyY"
                title="meetup-details"
                width="250"
        />
        <footer
                className="app-footer"
        >
                <img
                        alt="logo"
                        className="app-logo"
                        src="logo.svg"
                />
                <p>
                        A community project of 
                        <a
                                href="https://www.call-em-all.com/careers"
                                tabIndex={0}
                        >
                                Call-Em-All
                        </a>
                </p>
                <img
                        alt="logo"
                        className="app-logo-2"
                        src="logo.svg"
                />
        </footer>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <withStyles(TopNav) />,
          <MembershipTrend />,
          <withStyles(Attendance) />,
          <iframe
            frameBorder="0"
            height="485"
            src="http://meetu.ps/3fFbyY"
            title="meetup-details"
            width="250"
/>,
          <footer
            className="app-footer"
>
            <img
                        alt="logo"
                        className="app-logo"
                        src="logo.svg"
            />
            <p>
                        A community project of 
                        <a
                                    href="https://www.call-em-all.com/careers"
                                    tabIndex={0}
                        >
                                    Call-Em-All
                        </a>
            </p>
            <img
                        alt="logo"
                        className="app-logo-2"
                        src="logo.svg"
            />
</footer>,
        ],
        "className": "app",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "frameBorder": "0",
            "height": "485",
            "src": "http://meetu.ps/3fFbyY",
            "title": "meetup-details",
            "width": "250",
          },
          "ref": null,
          "rendered": null,
          "type": "iframe",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="logo"
                className="app-logo"
                src="logo.svg"
/>,
              <p>
                A community project of 
                <a
                                href="https://www.call-em-all.com/careers"
                                tabIndex={0}
                >
                                Call-Em-All
                </a>
</p>,
              <img
                alt="logo"
                className="app-logo-2"
                src="logo.svg"
/>,
            ],
            "className": "app-footer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo",
                "className": "app-logo",
                "src": "logo.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  "A community project of ",
                  <a
                    href="https://www.call-em-all.com/careers"
                    tabIndex={0}
>
                    Call-Em-All
</a>,
                ],
              },
              "ref": null,
              "rendered": Array [
                "A community project of ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Call-Em-All",
                    "href": "https://www.call-em-all.com/careers",
                    "tabIndex": 0,
                  },
                  "ref": null,
                  "rendered": "Call-Em-All",
                  "type": "a",
                },
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "logo",
                "className": "app-logo-2",
                "src": "logo.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "footer",
        },
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="app"
>
          <withStyles(TopNav) />
          <MembershipTrend />
          <withStyles(Attendance) />
          <iframe
                    frameBorder="0"
                    height="485"
                    src="http://meetu.ps/3fFbyY"
                    title="meetup-details"
                    width="250"
          />
          <footer
                    className="app-footer"
          >
                    <img
                              alt="logo"
                              className="app-logo"
                              src="logo.svg"
                    />
                    <p>
                              A community project of 
                              <a
                                        href="https://www.call-em-all.com/careers"
                                        tabIndex={0}
                              >
                                        Call-Em-All
                              </a>
                    </p>
                    <img
                              alt="logo"
                              className="app-logo-2"
                              src="logo.svg"
                    />
          </footer>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <withStyles(TopNav) />,
            <MembershipTrend />,
            <withStyles(Attendance) />,
            <iframe
              frameBorder="0"
              height="485"
              src="http://meetu.ps/3fFbyY"
              title="meetup-details"
              width="250"
/>,
            <footer
              className="app-footer"
>
              <img
                            alt="logo"
                            className="app-logo"
                            src="logo.svg"
              />
              <p>
                            A community project of 
                            <a
                                          href="https://www.call-em-all.com/careers"
                                          tabIndex={0}
                            >
                                          Call-Em-All
                            </a>
              </p>
              <img
                            alt="logo"
                            className="app-logo-2"
                            src="logo.svg"
              />
</footer>,
          ],
          "className": "app",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "frameBorder": "0",
              "height": "485",
              "src": "http://meetu.ps/3fFbyY",
              "title": "meetup-details",
              "width": "250",
            },
            "ref": null,
            "rendered": null,
            "type": "iframe",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="logo"
                  className="app-logo"
                  src="logo.svg"
/>,
                <p>
                  A community project of 
                  <a
                                    href="https://www.call-em-all.com/careers"
                                    tabIndex={0}
                  >
                                    Call-Em-All
                  </a>
</p>,
                <img
                  alt="logo"
                  className="app-logo-2"
                  src="logo.svg"
/>,
              ],
              "className": "app-footer",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "logo",
                  "className": "app-logo",
                  "src": "logo.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    "A community project of ",
                    <a
                      href="https://www.call-em-all.com/careers"
                      tabIndex={0}
>
                      Call-Em-All
</a>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  "A community project of ",
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Call-Em-All",
                      "href": "https://www.call-em-all.com/careers",
                      "tabIndex": 0,
                    },
                    "ref": null,
                    "rendered": "Call-Em-All",
                    "type": "a",
                  },
                ],
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "logo",
                  "className": "app-logo-2",
                  "src": "logo.svg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "footer",
          },
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
